src/styles/card.css

```diff
@@ -0,0 +1,100 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
```

Comment:
Will be better to set this style for all elements using `*` selector and move it to `normalize.css`